docs/src/dev/developer/release.asciidoc

```diff
@@ -75,11 +75,13 @@ might be high.
 .. `mvn verify -DskipPerformanceTests=false`
 . `bin/publish-docs.sh <username>` - note that under a release candidate the documentation is published as SNAPSHOT
 . `mvn versions:set -DnewVersion=xx.yy.zz -DgenerateBackupPoms=false` to update the project files to reference a non-SNAPSHOT version
+. `pushd gremlin-console/bin; ln -fs ../target/apache-tinkerpop-gremlin-console-xx.yy.zz-standalone/bin/gremlin.sh gremlin.sh; popd`
 . `git diff` and review the updated files (expect all `pom.xml` files and this README)
 . `git commit -a -m "TinkerPop xx.yy.zz release"` and `git push`
 . `git tag -a -m "TinkerPop xx.yy.zz release" xx.yy.zz` and `git push --tags`
 . `mvn clean install`
 . `mvn versions:set -DnewVersion=xx.yy.zz-SNAPSHOT -DgenerateBackupPoms=false` to go back to SNAPSHOT
+. `pushd gremlin-console/bin; ln -fs ../target/apache-tinkerpop-gremlin-console-xx.yy.zz-SNAPSHOT-standalone/bin/gremlin.sh gremlin.sh; popd`
```

Comment:
we bump versions in PMC VOTE:

https://github.com/apache/tinkerpop/blame/850e159b8dc883ac3995c8d775b8f09b7a64b440/docs/src/dev/developer/release.asciidoc#L143

and 

https://github.com/apache/tinkerpop/blame/850e159b8dc883ac3995c8d775b8f09b7a64b440/docs/src/dev/developer/release.asciidoc#L169

While we need those lines where you currently have them, we almost never execute those steps that you updated (i.e. we haven't done a release candidate since 3.0.0)